CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.0.1 (2020-07-08)
+
+- Moved `theming` to the `app` section in `skyuxconfig-schema.json` to match the corresponding TypeScript declarations. [#30](https://github.com/blackbaud/skyux-config/pull/30)
```

Comment:
The link here is wrong. I'm pretty sure it should point to https://github.com/blackbaud/skyux-config/pull/58. I'll go ahead and update.